src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8472,6 +8485,123 @@ paths:
+        Currently, this list may only ever contain zero or one ruleset entries as folders cannot use multiple rulesets.
+      operationId: listRulesets
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#parameters/nodeIdParam'
+        - $ref: '#/parameters/skipCountParam'
+        - $ref: '#/parameters/maxItemsParam'
+        - $ref: '#/parameters/rulesetMinimalEntryIncludeParam'
+      responses:
+        '200':
+          description: Successful response
+          schema: '#/definitions/RulesetPaging'
+        '400':
+          description: |
+            Invalid parameter: **nodeId** is not a valid format, or is not a folder
+        '401':
+          description: Authentication failed
+        '403':
+          description: User does not have permission to read rulesets on the node
+        '404':
+          description: |
+            **nodeId** does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/linked-rulesets':
+    post:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Link a ruleset
+      desciption: |
+        **Note:** this endpoint is available in Alfresco 7.3 and newer versions.
+        
+        Links a ruleset to the folder node **nodeId**.
+      operationId: linkRuleset
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#parameters/nodeIdParam'
+        - in: body
+          name: linkedRulesetBodyCreate
+          description: The ruleset to add.
+          required: true
+          schema:
+            $ref: '#/definitions/LinkedRulesetBodyCreate'
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/LinkedRulesets',
+        '400':
+          description: |
+            Invalid parameter: **nodeId** or **linkedRulesetBodyCreate** is invalid
+        '401':
+          description: Authentication failed
+        '403':
+          description: User does not have permission to create a link to this ruleset
+        '404':
+          description: |
+            **nodeId** or **id** does not exist
+        '409':
+          description: Another ruleset is already used by this folder
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/rulesets/{rulesetId}':
+    delete:
+      x-alfresco-since: "7.3.0"
+      tags:
+        - rules
+      summary: Removes the link between a ruleset and a folder
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.3.0 and newer versions.
+        
+        Remove the link between a ruleset and the folder **nodeId**.
+      operationId: removeLinkedRuleset
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - in: path
+          name: rulesetId
```

Comment:
Yes it does, I forgot to specify a description explaining the -default-

The rulesetId can be obtained through `GET /nodes/{nodeId}/rulesets` (since we agreed on using that endpoint for both owned and linked rulesets)